docs/source/getting_started/getting_started.md

```diff
@@ -18,32 +18,43 @@ We have two authentication tokens, one to connect to the back office and one to
 To authenticate, we need to set the `authenticationToken` header to one of them, depending on what we need to do.
 As our API expects all request bodies to be JSON encoded, we also need to set the `Content-Type` header to `application/json`.
 
+In the following examples, we are going to use three environment variables: `$URL`, `$BACKOFFICE_TOKEN`, and `$FRONTOFFICE_TOKEN`.
+To try the examples you need first to set them to the values for your system:
+
+```bash
+export URL="<url>"
+export BACKOFFICE_TOKEN="<backoffice_token>"
+export FRONTOFFICE_TOKEN="<frontoffice_token>"
+```
+
 # Ingest
 
 We can use the back office endpoint [`/documents`](https:/docs.xayn.com/back_office.html#operation/createDocuments) to ingest documents.
 
 We will ingest a document that represents this article: [https://xayn.com/blog/the-initial-challenge](https://xayn.com/blog/the-initial-challenge).
 
 ```bash
-curl -X POST https://<url>/documents
-    -H "authorizationToken: <back_office_token>"
-    -H "Content-Type: application/json"
+curl -X POST "$URL/documents" \
+    -H "authorizationToken: $BACKOFFICE_TOKEN" \
+    -H "Content-Type: application/json" \
     -d '{
```

Comment:
:art:/❓ throughout the examples the options are used in a mixed way sometimes as `-H`, `-d` and sometimes as `--header`, `--data`. we should use one style. for readability of the examples it may be better to use the long versions.

Reply:
I think we should use `--json ...` which is a combination of `--header "Content-Type: application/json"` and `--data ...`.

Except in one place where we use `--header`/`--data` and highlight that the `Content-Type` header is required and hint that `--json` does set it.

Reply:
Could be :pushpin: I think.